src/Analyzer/IQueryTreeNode.cpp

```diff
@@ -10,6 +10,8 @@
 
 #include <Parsers/ASTWithAlias.h>
 
+#include <magic_enum.hpp>
```

Comment:
Please create file `Common/EnumUtils.h` and hide this library under some function. For us it will be easier to track which features we use, and if we decide to switch to other reflection library.